core/modules/user/user.theme.inc

```diff
@@ -127,8 +127,10 @@ function theme_user_admin_permissions($variables) {
   foreach (element_children($form['role_names']) as $role_name) {
     $header[] = array('data' => backdrop_render($form['role_names'][$role_name]), 'class' => array('checkbox'));
   }
-  $output = '';
+  // Wrap the table in its own class so that we can make it scrollable.
```

Comment:
Can we please be specific by saying "horizontally scrollable" or "scrollable horizontally"?

Reply:
Yes.